package.json

```diff
@@ -103,9 +103,10 @@
     "@visx/brush": "^3.0.1",
     "@visx/legend": "^3.0.0",
     "@visx/pattern": "^3.0.0",
-    "@visx/responsive": "^2.10.0"
+    "@visx/responsive": "^2.10.0",
+    "eslint-plugin-typescript": "^0.14.0"
```

Comment:
did you mean to add this to the main dependencies?

Reply:
I did add it intentionally for testing TS. Since we're not using the plugin currently I've removed the package. Thanks!